modules/tinymce/src/core/main/ts/init/InitContentBody.ts

```diff
@@ -119,6 +118,7 @@ const mkSerializerSettings = (editor: Editor): DomSerializerSettings => {
       element_format: getOption('element_format'),
       entities: getOption('entities'),
       entity_encoding: getOption('entity_encoding'),
+      remove_trailing_brs: getOption('remove_trailing_brs'),
```

Comment:
Minor thing, but move this line before `url_convert_scope` since it is part of a `DomSerializerSettings` not `WriterSettings`.

Reply:
I'll move above `url_converter` for a alphabetic order